webpack/assets/javascripts/react_app/redux/actions/toasts/index.js

```diff
@@ -0,0 +1,32 @@
+import {
+  TOASTS_ADD,
+  TOASTS_HIDE,
+  TOASTS_DELETE
+  // TOASTS_STORE_IN_LOCALSTORAGE,
+  // TOASTS_GET_FROM_LOCALSTORAGE
+} from '../../consts';
+import store from '../../../redux';
+
+// provide mechanism to persist toasts to localStorage
+export const addToast = (toast) => {
+  const payload = Object.assign({}, toast, {visible: true});
+
+  // add to store
+  store.dispatch({ type: TOASTS_ADD, payload: payload });
+};
+
+export const hideToast = (id) => {
+  const payload = { id: id };
```

Comment:
That is my least favorite feature in ES6. I think it sacrifices clarity. I'll make the change though.